webpack/assets/javascripts/react_app/components/ConfigReports/Welcome.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { getManualURL, getWikiURL } from '../../common/helpers';
+
+export const WelcomeConfigReports = () => {
+  const content = __(`If you wish to configure Puppet to forward it reports to Foreman, 
+  please follow <a href=${getManualURL(
+    '3.5.4PuppetReports'
+  )}> setting up reporting   </a> and
+  <a href=${getWikiURL('Mail_Notifications')}> e-mail reporting </a>`);
```

Comment:
Again something odd in the formatting here, could be eslint's fault. No need for spaces inside the `<a>` tags as well, or the underlines might display also for the spaces:
```suggestion
  const content = __(`If you wish to configure Puppet to forward it reports to Foreman, 
    please follow <a href=${getManualURL('3.5.4PuppetReports')}>setting up reporting</a> and
    <a href=${getWikiURL('Mail_Notifications')}>e-mail reporting</a>`);
```